dbschema/variance-explanation.esdl

```diff
@@ -0,0 +1,16 @@
+module ProgressReport {
+  type VarianceExplanation {
+    required report: default::ProgressReport {
+      readonly := true;
+    };
```

Comment:
This needs to be an `Engagement::Child`.
It probably makes sense to have a `ProgressReport::Child` as well following suit since we'll have several of these.

Reply:
@bryanjnelson or @CarsonF, I'll need some help on this one.  I extended the VarianceExplanation type as an Engagement::Child, but not sure if that is correct, or how to implement your feedback.